insights/parsers/cluster_conf.py

```diff
@@ -1,167 +1,15 @@
+"""
+ClusterConf - file ``/etc/cluster/cluster.conf``
+================================================
+
+Stores a filtered set of lines from the cluster config file.  Because of the
+filtering, the content as a whole will not parse as XML.  We use a
+:class:`insights.core.LogFileOutput` parser class because, sadly, it's
+easiest.
+"""
 from .. import LogFileOutput, parser
-import xml.etree.ElementTree as ET
 
 
 @parser("cluster.conf")
 class ClusterConf(LogFileOutput):
```

Comment:
@PaulWay please add docstrings to this class so that it will be included in the documentation.

Reply:
Stupid of me to forget :-)  Thanks, @bfahr, have done so.